googet.go

```diff
@@ -466,7 +466,7 @@ func main() {
 	cmdr.ImportantFlag("verbose")
 	cmdr.ImportantFlag("noconfirm")
 
-	nonLockingCommands := []string{"help", "commands", "flags"}
+	nonLockingCommands := []string{"help", "commands", "flags", "listrepos"}
 	if ggFlags.NArg() == 0 || goolib.ContainsString(ggFlags.Args()[0], nonLockingCommands) {
```

Comment:
You will need to move the check on 474 above this as listrepos needs rootDir to be set

Reply:
Done,
I've attempted to maintain the existing logic to allow help&commands to proceed as before